crates/astria-sequencer/src/mint/action.rs

```diff
@@ -47,8 +48,11 @@ impl ActionHandler for MintAction {
             .get_account_balance(self.to, native_asset)
             .await
             .context("failed getting `to` account balance")?;
+        let new_balance = to_balance
+            .checked_add(self.amount)
+            .ok_or_else(|| anyhow!("`to` balance would overflow"))?;
```

Comment:
`anyhow` (different from `eyre`) implements `Context for Option<T>`:
```suggestion
            context("`to` balance would overflow")?;
```

Reply:
Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).